LoRaEngine/test/LoRaWanNetworkServer.Test/MessageProcessor_End2End_NoDep_CloudToDeviceMessage_SizeLimit_Should_Accept.cs

```diff
@@ -97,7 +97,7 @@ public class MessageProcessor_End2End_NoDep_CloudToDeviceMessage_SizeLimit_Shoul
 
             this.LoRaDeviceClient.SetupSequence(x => x.ReceiveAsync(It.IsAny<TimeSpan>()))
                 .ReturnsAsync(cloudToDeviceMessage)
-                .ReturnsAsync(null);
+                .ReturnsAsync<Message>((Message)null);
```

Comment:
I think you can do either ReturnsAsync<Message>(null) or ReturnsAsync((Message)null)